miso-web/src/main/webapp/pages/editProject.jsp

```diff
@@ -502,7 +516,7 @@
   Studies
 <div id="studies_section_arrowclick" class="toggleLeft"></div>
```

Comment:
Some of these point left when the section is expanded and down when collapsed. Some are the opposite. I think they should all be pointing down when collapsed and left when expanded (though up would make more sense)... :man_shrugging:

Reply:
Updated. Regrettably, we have no up arrow in the icons...

Reply:
I would vote for + and -, since that's much clearer, but not relevant to this PR.